examples/jupyter/tutorials/tutorial3-python-vector.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# libCEED for Python examples\n",
+    "\n",
+    "This is a tutorial to illustrate the main feautures of the Python interface for [libCEED](https://github.com/CEED/libCEED/), the low-level API library for efficient high-order discretization methods developed by the co-design [Center for Efficient Exascale Discretizations](https://ceed.exascaleproject.org/) (CEED) of the [Exascale Computing Project](https://www.exascaleproject.org/) (ECP).\n",
+    "\n",
+    "While libCEED's focus is on high-order finite/spectral element method implementations, the approach is mostly algebraic and thus applicable to other discretizations in factored form, as explained in the [user manual](https://libceed.readthedocs.io/) and [Doxygen documentation](https://codedocs.xyz/CEED/libCEED/)."
+   ]
+  },
+  {
```

Comment:
Yup, I wasn't sure if we wanted these tutorials to be self-contained, so that, let's say, one can pull up only a single tutorial and give a live demo in a workshop type of setting, without having to remember "oh, I had to run the first notebook first"

Reply:
I like to try to reduce duplication, but either is fine for me. I just wanted to make sure we were making a conscious decision